.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Add blank line at end of file

Reply:
Marking resolved since this is now out of scope.

Reply:
Done